dashboard/app/models/school_info.rb

```diff
@@ -73,6 +74,51 @@ def school_state=(input)
     end
   end
 
+  # Only sync from school on create to avoid unintended updates to old data
+  before_validation :sync_from_schools, on: :create
+
+  def sync_from_schools
+    # If a SchoolInfo is linked to a School then the SchoolInfo pulls its data from the School
+    # It seems like there is some code that is passing in mismatched data at times.
+    # As of Nov. 2, 2017 there were 7 rows in school_infos with non-null school_id and conflicting
+    # data between schools and school_infos. Until we better understand what is causing that we don't want
+    # to flat out fail if we get conflicting inputs. Instead we overwrite the passed in fields with what
+    # is on the School and report the mismatch to HoneyBadger.
+    unless school.nil? && school_id.nil?
+      original = {}
+      original[:country] = country
+      original[:school_type] = school_type
+      original[:state] = state
+      original[:zip] = zip
+      original[:school_district_id] = school_district_id
+      original[:school_district_other] = school_district_other
+      original[:school_district_name] = school_district_name
+      original[:school_other] = school_other
+      original[:school_name] = school_name
+      original[:full_address] = full_address
+      original[:validation_type] = validation_type
```

Comment:
This can be 
```ruby
original = attributes.slice *%w(
  country
  school_type
  state
)
```
etc with all the fields

Reply:
I considered something like that but it was about the same amount of code and this way  felt more readable to me.